### ISSUE TITLE
[REQ] Distribute versions on Homebrew

### ISSUE THREAD

**Issue by rhys-rantmedia** [OpenAPITools/openapi-generator#11816]
### Is your feature request related to a problem? Please describe.

I have encountered [this bug](https://github.com/OpenAPITools/openapi-generator/pull/11202#issuecomment-1060543691) in OpenAPI Generator 5.4.0, which isn't present in 5.3.1 due to the change in the `JSONEncodable` code. There seems to also be a bugfix in the master branch 6.0.X build. Installing openapi-generator 5.3.1 via Homebrew is a longwinded though necessary process for this bug fix.

## Describe the solution you'd like

I would like to be able to install multiple versions of OpenAPI Generator through Homebrew, using the syntax `brew install openapi-generator@5.3.1`

## Describe alternatives you've considered

Reinstalling an old version of openapi-generator using Homebrew is an involved process that requires:
  - finding a commit before openapi-generator was updated
  - reverting Homebrew's formula list to that commit
  - potentially uninstalling dependencies if you have a more recent version installed than that version of Homebrew recognises
  - installing openapi-generator at that version
  - reverting the git changes to your local Homebrew install
  - pinning openapi-generator so that the installation doesn't again get overwritten by the updated version

**Comment by rhys-rantmedia** [comment]
> What about using https://www.npmjs.com/package/@openapitools/openapi-generator-cli to manage openapi-generator cli version instead?

Wasn't aware that was a thing!
Also, I mostly try to keep my software installed using Homebrew and avoid npm where possible. But that's a me thing, so I understand that this request is either very low prio/won't do if there's already a solution available. Still would be nice to have though! 😄

**Comment by sanfordj** [comment]
agreed, homebrew formulas for the specific versions would be appreciated.
